packages/features/ee/workflows/lib/reminders/templates/emailReminderTemplate.ts

```diff
@@ -45,11 +47,13 @@ const emailReminderTemplate = (
 
   const attendeeHtml = `<div><strong class="editor-text-bold">Attendees: </strong></div>You & ${otherPerson}<br><br>`;
 
+  const locationHtml = `<div><strong class="editor-text-bold">Location: </strong></div>${location}<br><br>`;
+
```

Comment:
This should be like this:-
![Screenshot 2024-08-14 135234](https://github.com/user-attachments/assets/83844b7c-287d-461d-95e6-cc908ecda4b6)